metadata_annotation.ipynb

```diff
@@ -0,0 +1,2166 @@
+{
```

Comment:
<p>Can you explain what you're trying to say here, please? I don't get the context.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>